### ISSUE TITLE
2 Way Audio not working in Frigate Card

### ISSUE THREAD

**Issue by Daz42** [pergolafabio/Hikvision-Addons#184]
### What happened?

Not an issue with the Hikvision Camera Integration - however I was unable to consistently get 2 way audio from the Frigate card for the Hikvision Camera Integration. This is an issue being dealt with but in the meantime we are using the web-rtc card which works but slightly clunky.

### Add-on

Hikvision Doorbell

### What version of the add-on are you running?

3.0.14

### Installation type

Home Assistant add-on

**Comment by Daz42** [comment]
I would like to add a guide for those wanting to get their Hikvision doorbell working with the web-rtc card complete with mp3 playback. Eventually it will be updated for the Frigate card when 2 way audio is resolved with it.
[Home Assistant Hikvision Doorbell (for web) As-Built v1.0.docx](https://github.com/pergolafabio/Hikvision-Addons/files/14169277/Home.Assistant.Hikvision.Doorbell.for.web.As-Built.v1.0.docx)

**Comment by pergolafabio** [comment]
Great guide , not sure if I need to combine it or not... Gonna wait for the frigate updates first, because that card is nicer indeed... Also next update doesn't require frigate add-on/integration...

Just question, why do you have the camera defined 2 times in go2rtc , why not use the first one for the mp3 also?

**Comment by Daz42** [comment]
I tried using the first one but could not get the mp3 to play for the life of me. When I defined it again but with channel 102 it works reliably every time. Also another weirdness, when I update webrtc-camera to version 3.5.1 then I start having issues with mp3 playback and intermittent 2 way audio. I'm on master which is working but obviously should move on once I find out why it's playing up!

**Comment by TokarevSergey** [comment]
You managed to solve the problem. I have a Hikvision camera and doorbell. The problem was this. At first there was sometimes intermittent two-way communication. And now when I press the microphone, the sound disappears.

**Comment by pergolafabio** [comment]
ok,  found the issue!!

https://github.com/dermotduffy/frigate-hass-card/issues/1356
fixed in this:
https://github.com/AlexxIT/go2rtc/pull/967
https://github.com/AlexxIT/go2rtc/issues/962

The issue that two way audio isnt working the second time, is that the "close" command is not being used in the go2rtc addon, it will be implemlented later!!

for now i send the command below, right after you do the answer/hangup and before pressing the mic button, this makes the previous twoway audio close first...

`"curl -i --digest -u admin:XXXX-X PUT  http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close"`

I keep this one open, untill a new go2rtc addon is released!!

**Comment by TokarevSergey** [comment]
Thank you

**Comment by TokarevSergey** [comment]
Thank you. I'm really looking forward to the fix.

**Comment by pergolafabio** [comment]
Same, but the curl is working great!!

**Comment by TokarevSergey** [comment]
> ок, нашел проблему!!
> 
> [dermotduffy/frigate-hass-card#1356](https://github.com/dermotduffy/frigate-hass-card/issues/1356) исправлено здесь: [AlexxIT/go2rtc#967 ](https://github.com/AlexxIT/go2rtc/pull/967) [AlexxIT/go2rtc#962](https://github.com/AlexxIT/go2rtc/issues/962)
> 
> Проблема, из-за которой двусторонняя аудиосвязь не работает во второй раз, заключается в том, что команда «закрыть» не используется в аддоне go2rtc, она будет реализована позже!
> 
> на данный момент я отправляю команду ниже, сразу после того, как вы ответите/повесите трубку и перед нажатием кнопки микрофона, это сначала закроет предыдущую двустороннюю аудиосвязь...
> 
> `"curl -i --digest -u admin:XXXX-X PUT http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close"`
> 
> Я оставляю этот вопрос открытым, пока не выйдет новый аддон go2rtc!!

Добрый вечер не могу заставить это работать.
У вас получилось.
Я делаю в файле configuration.yaml запись 
shell_command:
  doorbelltvaoff: curl -i --digest -u admin:ST008855h PUT  http://10.0.0.11/ISAPI/System/TwoWayAudio/channels/1/close
и запускаю ее с помошью скрипта звук вске равно прерывистый. Подскажите может я чтото не так понял.

**Comment by pergolafabio** [comment]
I do this in the frigate card for my "phone/answer" button:

```
            - type: custom:frigate-card-menu-icon
              icon: mdi:phone
              tap_action:
                - action: call-service
                  service: script.answer_call
                - action: custom:frigate-card-action
                  frigate_card_action: unmute
                - action: custom:frigate-card-action
                  frigate_card_action: microphone_unmute
```

First i do the script, to make my answer+hangup+close audio
Then i do the frigate unmute functions for mic/audio

This is my script:

```
answer_call:
  alias: Deur open answer call
  sequence:
  - service: button.press
    entity_id: button.ds_kh9510_answer_call
  - delay: 
      milliseconds: 500
  - service: button.press
    entity_id: button.ds_kh9510_hangup_call
  - service: shell_command.hikvision_close_audio
```

and my shell command is:

`hikvision_close_audio: "curl -i --digest -u admin:XXXXX-X PUT  http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close"`

**Comment by TokarevSergey** [comment]
> hikvision_close_audio: "curl -i --digest -u admin:XXXXX-X PUT http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close"

hikvision_close_audio: "curl -i --digest -u admin:XXXXX -X PUT  http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close"

**Comment by TokarevSergey** [comment]
> правильный

Блин ничего не получается. У меня все равно звук становится тише и потом начинает прирыватся если я несколько раз войду зайду на карточьку с микрофоном.

**Comment by TokarevSergey** [comment]
> I do this in the frigate card for my "phone/answer" button:
> 
> ```
>             - type: custom:frigate-card-menu-icon
>               icon: mdi:phone
>               tap_action:
>                 - action: call-service
>                   service: script.answer_call
>                 - action: custom:frigate-card-action
>                   frigate_card_action: unmute
>                 - action: custom:frigate-card-action
>                   frigate_card_action: microphone_unmute
> ```
> 
> First i do the script, to make my answer+hangup+close audio Then i do the frigate unmute functions for mic/audio
> 
> This is my script:
> 
> ```
> answer_call:
>   alias: Deur open answer call
>   sequence:
>   - service: button.press
>     entity_id: button.ds_kh9510_answer_call
>   - delay: 
>       milliseconds: 500
>   - service: button.press
>     entity_id: button.ds_kh9510_hangup_call
>   - service: shell_command.hikvision_close_audio
> ```
> 
> and my shell command is:
> 
> `hikvision_close_audio: "curl -i --digest -u admin:XXXXX-X PUT http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close"`

Так какже все таки правильно?

**Comment by pergolafabio** [comment]
yeah thats why i do

**Comment by TokarevSergey** [comment]
> admin:XXXXX-X PUT

 admin:XXXXX-X PUT

**Comment by pergolafabio** [comment]
it works, but if you answer the doorbell 2 times when not refreshing the card, i dont have audio, you can send the close command i posted earlier

i saw new updates on the card, didnt try yet, maybe its better now

**Comment by pergolafabio** [comment]
sorry, no idea, for mic to work, you need https with valid ssl, mauybe thats the issue?

**Comment by spn91** [comment]
Hi, I have the same problem with two-way-audio when browsing away from the frigate-card after first use of the microphone. The first time it works, but if I browse away to another lovelace section and then back to the section with the frigate-card, two-way-audio isn't working anymore. The browser doesn't show the microphone status icon again, so for me it seems that no new microphone connection is made to the browser?

Below is the relevant part of the frigate-card config, I thought using the ISAPI channel close command would solve my problem, but it isn't. I have to reload the whole page, or wait a few minutes. Then the mic connections starts again if I click the phone icon. 

```
- type: custom:frigate-card-menu-icon
    icon: mdi:phone
    tap_action:
      - action: custom:frigate-card-action
        frigate_card_action: unmute
      - action: custom:frigate-card-action
        frigate_card_action: microphone_unmute
      - action: perform-action
        data:
          entity_id: button.eingang_hangup_call
        perform_action: button.press
      - action: call-service
        service: text.set_value
        data:
          entity_id: text.eingang_isapi_request
          value: PUT /ISAPI/System/TwoWayAudio/channels/1/close
```

Any idea? Also in the addon log I can see that the channel close command works successfully 

```
2024-12-29 18:57:27.743 | DEBUG    | sdk.utils:call_ISAPI:165 - Response output: {
	"statusCode":	1,
	"statusString":	"OK",
	"subStatusCode":	"ok"
}
```

**Comment by pergolafabio** [comment]
When I tested it a long time ago, I saw the phone button always,  I was able to toggle it...
Maybe something changed in the frigate card? I saw a lot of updates on it,... Didn't test it anymore...

**Comment by spn91** [comment]
Hmm I always see the phone button? Do you mean the microphone button? I always see the microphone button in the frigate-card. I have the same problem as you had, until you were able to fix it with the channel close command. 

What I meant with 
> The browser doesn't show the microphone status icon again

is, that my browser shows the red microphone icon on the tab which is using the microphone. This icon is missing.

**Comment by pergolafabio** [comment]
Hmm, not sure, I didn't test it anymore for a year :-(

**Comment by spn91** [comment]
Hmm, than I don't understand your comment you made here, only a few months ago 😅  -> https://github.com/dermotduffy/frigate-hass-card/issues/1356#issuecomment-2373033786

**Comment by pergolafabio** [comment]
Well, if I do the hangup, I send the close command also, so the twowayaudio is really closed..
Next time a on a new ring, if I answer and start two way audio I can hear audio again... With this approach I didn't need to hard refresh the Lovelace card

**Comment by spn91** [comment]
I also send to close command now when pressing the phone-hangup button. Still not working when 

would you be so kind as to share your current config of the card?

**Comment by pergolafabio** [comment]
I don't use it anymore, the last update is on the GitHub page :-)
I stopped using it before the frigate card 6.x came out...

**Comment by spn91** [comment]
just curios, what are you using now? the webrtc card?

**Comment by pergolafabio** [comment]
at the moment, hikconnect :-)
i just wrote some guides/tutorials to help others starting...

The problem with HA on my android is that the android push notification is not always instant when, this is very imortant for a  doorbell offcourse, i see hikconnect notification coming in faster ...

but i will try to test the card again in coming days, i need to change the guide anwyay, since its was written for the frigate addon, that addon is not needed anymore, i believe also some service code was changed...

**Comment by pergolafabio** [comment]
quicly tested it, there is indeed something strange now with the card, i didnt see that behaviour before
for the original mic buttons, i can indeed only press it once, that it seems to freeze , even closing the browser doesnt help? 

anyway, i looked again at your code, for the custom "phone" button; you dont have an indoor panel right? but you are only sending the "hangup" command? in order to make the "hangup" work, you first need to "answer"
so try first sending the "answer" that makes the doorbell stop ringing, then you do the "hangup", to "stop" the real call , then send the "close audio", so the audio is really closed
then start two way audio ....


example here, in my case with indoor, since i need to send the answer/hangup to indoor and close audio to outdoor...

```
            - type: custom:frigate-card-menu-icon
              icon: mdi:phone
              tap_action:
                - action: call-service
                  service: button.press
                  data:
                    entity_id: button.ds_kh9510_answer_call
                - action: custom:frigate-card-action
                  frigate_card_action: sleep
                  duration:
                    ms: 500
                - action: call-service
                  service: button.press
                  data:
                    entity_id: button.ds_kh9510_hangup_call
                - action: call-service
                  service: text.set_value
                  data:
                    entity_id: text.ds_kd8003_isapi_request
                    value: PUT /ISAPI/System/TwoWayAudio/channels/1/close
                - action: custom:frigate-card-action
                  frigate_card_action: unmute
                - action: custom:frigate-card-action
                  frigate_card_action: microphone_unmute
```

**Comment by pergolafabio** [comment]
i also see now this circle, something is loading in background, and its not able to load...
do you see that too?

![image](https://github.com/user-attachments/assets/a1720b76-bbe6-447b-9fdd-815917fa8f71)

**Comment by pergolafabio** [comment]
seems an ongoing issue?

https://github.com/dermotduffy/frigate-hass-card/issues/1773

**Comment by spn91** [comment]
Ok, I'm using iOS and for me the push comes in nearly instant.

Alright, your documentation worked really well for me. Good to know that the frigate addon isn't needed anymore. I assume that I have to use the go2rtc addon?

Hangup via your hikvision addon sent to my outdoor station works for me. I don't need to answer it first. When I send the hangup event, the call on the outdoor station stops. Maybe this works for me, because my doorbell calls an extension which is  a queue, so it is already in the call when I send the command.

The circle means that your webrtc connection is not working and the frigate card instead uses an image of the stream every 1 second. For me this was the case, when connection from outside of my network. To fix this, I had to forward port 8555 to my HA instance. Inside my network everything was working fine from the beginning.

**Comment by pergolafabio** [comment]
hmm, need to invstogate; but the stream is working for me still with that blue circle.... i see the video changing every xx sec

**Comment by pergolafabio** [comment]
crap, you are right, i removed the go2rtc config :), cause i didnt use anymore ...

leet me try again later today

**Comment by spn91** [comment]
Yeah, and with the blue circle you also have no mic icon, because this is just a image which is delivered via HA camera proxy as a fallback and to have an image while the real stream loads in the background.

If you get everything working, could you please try the following steps?

1. Activate mic on frigate-card on for example a browser tab. 
2. Hang-up the call
3. Try to activate the mic on another device and speak.

For me this is not working (no voice output, or some really distorted voice) , because somehow the connection on the browser tab is not closed (browser mic icon still showing on the tab). It works again if I reload the browser tab, or wait for 90 seconds (this is the default value for closing the mic connection which is used for the frigate-card if nothing else is set).

**Comment by pergolafabio** [comment]
ok, got my go2rtc working again, but the card is indeed giving me an headache! i saw the mic button once, i was able to start two way audio ... now the mic button is gone , even refresh doenst help ...
when i browse directly to the go2rtc stream page it works, i use https, my setup is correct with go2rtc

**Comment by pergolafabio** [comment]
the blue circle is now gone, i can see a fluent stream, but no mic button anymore , not sure why :-(

this is my config:

```
            - camera_entity: camera.deurbel
              live_provider: go2rtc
              go2rtc:
                url: https://go2rtc:XXXXXX@go2rtc.XXXXX:8123
                stream: deurbel
                modes:
                  - webrtc
```

using that url stream in a chrome browser, from external works everytime with chrome, but the card doesnt , not sure why

**Comment by pergolafabio** [comment]
ahh, maybe i know why, maybe its because i use camera.deurbel , and its also now using go2rtc in the core ... so its a double go2rtc now, and is conflicting with the go2rtc in the core.... since we need to use the addon ...

ok, gonna change that now

**Comment by pergolafabio** [comment]
ok, got my mic buton back :-) 

need to the real testing later :-)

**Comment by pergolafabio** [comment]
ok, updated my guide
https://github.com/pergolafabio/Hikvision-Addons/blob/main/frigate/twowayaudio.with.frigate.md

i added the close isapi command to the hangup button instead :-)

**Comment by spn91** [comment]
Nice 👍 

Could you please test the steps described in https://github.com/pergolafabio/Hikvision-Addons/issues/184#issuecomment-2565236847 ?

**Comment by pergolafabio** [comment]
on another device you mean for the second call ?

**Comment by spn91** [comment]
No calling, just open the frigate card on device 1, activate mic, click hangup button. Open frigate card on device 2 and activate mic (within 90 seconds). Is the audio working now flawlessly?

**Comment by pergolafabio** [comment]
there is audio/mic everytime, thats not an issue, but its a cracky sound , but thats maybe because of the android itself
but i always have twoway audio yes

**Comment by spn91** [comment]
No, the cracky sound is (I think) because of the two connections. I have this cracky sound on every device which connects as second device. At least this is a "problem" of the frigate-card. Although the mic is muted, the two-way audio stream still stays connected. It would be nice to have an option to reset the stream back to 1-way-audio (which could be used during the hangup action).


Could you please try something else? 

1. Start two-way-audio with frigate-card
2. hang-up the "call"
3. browse to another lovelace section on your dashboard
4. browse back to the lovelace section which contains the frigate-card
5. try to start two-way-audio again

Does this work for you? In my case the mic is not activated again.

**Comment by pergolafabio** [comment]
yes, that works, but ONLY if i also send the command below when i  click on the hangup button (or send if before you you do do the answer) ... thats why the "original" mic button on the card doesnt work, it doesnt send the isapi command
if i dont send that command, then indeed the second time the twoway audio doesnt work, because i believe its still in use on the first session... so browssing to another page and back gives issues indeed!! Only a hard refresh or close/open companion app does the trick!

maybe the command below on your doorbell station doesnt work? whay happens if you do it in putty? gives it back an 200 state?

shell:
`curl -i --digest -u admin:XXX-X PUT  http://192.168.0.70/ISAPI/System/TwoWayAudio/channels/1/close`

```
                - action: call-service
                  service: text.set_value
                  data:
                    entity_id: text.ds_kd8003_isapi_request
                    value: PUT /ISAPI/System/TwoWayAudio/channels/1/close
```

**Comment by pergolafabio** [comment]
> It would be nice to have an option to reset the stream back to 1-way-audio (which could be used during the hangup action).

isnt the action below forcing it back to oneway audio?

```
              tap_action:
                - action: custom:frigate-card-action
                  frigate_card_action: microphone_mute
```

**Comment by spn91** [comment]
I do send the close command. In your addon I can see the reply, which contains "ok". So the command seems to work. I'm not able to try with curl, as I have a "!" in my password and I wasn't able to get it working, though trying different methods of escaping it..

The `frigate_card_action: microphone_mute` just mutes the mic, but doesn't revert back the stream. I can easily see this, because the browser still shows the mic indicator after pressing the button.


My frigate-card config
<details>

```yaml
type: custom:frigate-card
cameras:
  - camera_entity: camera.doorbell
    live_provider: go2rtc
    go2rtc:
      modes:
        - webrtc
menu:
  style: outside
  position: bottom
  buttons:
    microphone:
      enabled: true
      type: toggle
    screenshot:
      enabled: false
    download:
      enabled: false
    fullscreen:
      enabled: false
    snapshots:
      enabled: false
    timeline:
      enabled: false
    media_player:
      enabled: false
    clips:
      enabled: false
    live:
      enabled: false
    cameras:
      enabled: false
    frigate:
      enabled: false
    camera_ui:
      enabled: false
live:
  controls:
    builtin: true
  auto_mute: []
  lazy_unload: []
  microphone:
    always_connected: false
    disconnect_seconds: 20
elements:
  - type: custom:frigate-card-menu-icon
    icon: mdi:volume-high
    tap_action:
      - action: custom:frigate-card-action
        frigate_card_action: unmute
  - type: custom:frigate-card-menu-icon
    icon: mdi:volume-off
    tap_action:
      - action: custom:frigate-card-action
        frigate_card_action: mute
  - type: custom:frigate-card-menu-icon
    icon: mdi:phone
    tap_action:
      - action: custom:frigate-card-action
        frigate_card_action: unmute
      - action: custom:frigate-card-action
        frigate_card_action: microphone_unmute
      - action: perform-action
        data:
          entity_id: button.eingang_hangup_call
        perform_action: button.press
      - action: call-service
        service: text.set_value
        data:
          entity_id: text.eingang_isapi_request
          value: PUT /ISAPI/System/TwoWayAudio/channels/1/close
  - type: custom:frigate-card-menu-icon
    icon: mdi:phone-hangup
    tap_action:
      - action: custom:frigate-card-action
        frigate_card_action: microphone_mute
      - action: call-service
        service: text.set_value
        data:
          entity_id: text.eingang_isapi_request
          value: PUT /ISAPI/System/TwoWayAudio/channels/1/close
  - type: custom:frigate-card-menu-icon
    icon: mdi:door-open
    hold_action:
      - action: perform-action
        data:
          entity_id: switch.eingang_door_relay_0
        perform_action: switch.turn_on
dimensions:
  aspect_ratio_mode: static
  aspect_ratio: "16:9"
cameras_global:
  dimensions:
    layout:
      fit: fill
status_bar:
  style: none
```

</details>

**Comment by pergolafabio** [comment]
But you are using the core go2rtc config? Maybe that's conflicting? I had that too , I removed the camera. ...
Forcing using the go2rtc addon instead of the core...

Have a look at my code again, I updated it, I use an URL instead...

**Comment by pergolafabio** [comment]
The core integration doesn't support two way audio

**Comment by spn91** [comment]
No, the frigate-card example was using go2rtc via the frigate addon.

**Comment by pergolafabio** [comment]
Ah ok, I don't use that frigate addon

**Comment by spn91** [comment]
At the moment I'm trying to get rid of the frigate addon. go2rtc addon already works, but I want to use the core go2rtc integration which has the option to specify an go2rtc server -> https://www.home-assistant.io/integrations/go2rtc/#examples

**Comment by pergolafabio** [comment]
Are you running the latest go2rtc binary,? They added some changes to it, maybe the frigate addon isn't using the latest go2rtc

**Comment by pergolafabio** [comment]
There was an audio close channel command added to the go2rtc addon, maybe since you are using the frigate addon, you are missing that one, I look up later

**Comment by pergolafabio** [comment]
https://github.com/AlexxIT/go2rtc/pull/967

**Comment by pergolafabio** [comment]
Btw, I don't think you need to send the Isapi close command at all... It's only needed when you send first the answer command, in my setup, the doorbell has actually the audio open ... If you send just hangup, there is no need to close audio

**Comment by spn91** [comment]
Frigate is using a version > 1.9.0, so the fixed version should be already included.
But forget Frigate.

I have the go2rtc addon running. I don't want to expose go2rtc on my proxy. I tried to use the go2rtc stream url which I get when visiting the go2rtc UI via HA Ingress, but this is not working in the frigate-card (stream is not starting, no request seen in the go2rtc log).

Now there is the core go2rtc integration (https://www.home-assistant.io/integrations/go2rtc/#examples) where I have specified my go2rtc addon address, but nothing happens (no error, no new entity, just nothing). Do you know what additional steps I have to take to get the camera into HA?

**Comment by pergolafabio** [comment]
ingress indeed doesnt work i think, but i use nginx, i didnt expose 1984 or 1985 ; i just use 8123 to access go2rtc , but offcourse its open to outside word, so used a strong user/pass combo
i believe thats the only way to make it work... i saw some new updates on the frigate card, that it can also use an builtin proxy, like the webrtc card, but didnt test that yet

**Comment by pergolafabio** [comment]
not sure, you still need to point to the go2rtc addon, but i didnt test that yet

will do it in a few moments :)

**Comment by pergolafabio** [comment]
this is what i was talking about also:

https://github.com/dermotduffy/frigate-hass-card/issues/1299

**Comment by pergolafabio** [comment]
if you use this:
```
go2rtc:
  url: http://my-go2rtc-instance:1984
```

i dont think it will work, since its not running on https, for the mic support you need https .. that means go2rtc needs to be running with a valid ssl anyway?

**Comment by pergolafabio** [comment]
its a way of disabling the core integration if you dont need it ...

i use the core integration for normal cameras, and i use the addon only for my tuya cameras, they are quite specific and dont work with the core... maybe some users dont want to use the core at all

**Comment by pergolafabio** [comment]
but let me try, rebooting HA now, specified the addon now with the core integration

**Comment by spn91** [comment]
Do you have any expectation what should happen after specifying the external go2rtc server? I thought it will create a camera entity.

**Comment by pergolafabio** [comment]
hmm, that doesnt work, specified the addon url, then did it like below, but it fails to load, it needs https

```
        - type: custom:frigate-card
          cameras:
            - live_provider: go2rtc
              go2rtc:
                url: http://user:pass@192.168.0.17:8123
                stream: deurbel
                modes:
                  - webrtc
```

**Comment by pergolafabio** [comment]
> Do you have any expectation what should happen after specifying the external go2rtc server? I thought it will create a camera entity.

no, it doesnt create an camera entity, its just if you want to make control of your own go2rtc instance, and dont want to make use of the core integration at all, you still need to make the cameras in go2rtc.yaml, but even that doesnt create an camera enitty... i'm not pointing to a camera entity in my frigate card, i actually point to the stream source from go2rtc

**Comment by spn91** [comment]
I think I don't understand the use case of using the external go2rtc server with the core integration. 
What would be the next steps after specifying the url for me? At the moment I have no camera entity for the doorbell.

I know that you have specified the url in the frigate-card, but I thought that I can create the entity in HA and then using this entity (just like before using the frigate integration).

**Comment by spn91** [comment]
I think I have found out for what it is used for.. I have created a generic.camera in HA and suddenly this camera also was present in the go2rtc addon, without specifying it in the go2rtc.yml config. So the core go2rtc integration pushes every camera to the go2rtc addon it seems.

**Comment by spn91** [comment]
got it working now with go2rtc addon and frigate-card, but issue still exists. If I browse away on the lovelace card and back to it, somehow the mic session is lost and the microphone won't work.. :(

**Comment by pergolafabio** [comment]
thats annoying :-(
do you use browser_mod ? if believe that integration, you can reload/refresh the page with an action, maybe you can use that service before starting the talk ?

**Comment by pergolafabio** [comment]
gonna close this one, since the close audio command is a good workaround for now
